sycl/test/CMakeLists.txt

```diff
@@ -94,7 +94,7 @@ add_lit_testsuite(check-sycl-dumps "Running ABI dump tests only"
   EXCLUDE_FROM_CHECK_ALL
   )
 
-if(SYCL_BUILD_PI_CUDA)
+if(SYCL_BUILD_CUDA)
```

Comment:
Just an observation: I don't think that `sycl/test` should be repeated for different triples. If there are some integration tests which check some intermediates produced by the compiler and they depend on the triple - they should set it explicitly.

Otherwise, tests which are supposed to be device-agnostic should not be run for each triple individually.

I created #14671 to track this.